examples/migrate_forseti/tutorial.md

```diff
@@ -0,0 +1,362 @@
+
+Update the <walkthrough-editor-select-regex
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  regex="PROJECT_ID">project_id</walkthrough-editor-select-regex>
+to match your chosen project_id.
+
+Update the <walkthrough-editor-select-regex
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  regex="RESOURCE_NAME_SUFFIX">resource_name_suffix</walkthrough-editor-select-regex>
+to match the string appended to the GCP resources used by Forseti.  For example, this string can be found appended to the name of either the Forseti Server or Forseti Client VM's.
+
+Update the <walkthrough-editor-select-regex
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  regex="ORG_ID">org_id</walkthrough-editor-select-regex>
+to match your organization id.
+
+### Choose Network
+Update the <walkthrough-editor-select-regex
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  regex="default">network</walkthrough-editor-select-regex>
+you wish to deploy Forseti in.
+
+Update the <walkthrough-editor-select-line
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  startLine=51
+  endLine=51
+  startCharacterOffset=21
+  endCharacterOffset=28>subnetwork</walkthrough-editor-select-line>.
+
+
+### Shared VPC
+If using a shared VPC, you'll need to update the related input variables.
+
+Update the <walkthrough-editor-select-regex
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  regex="SHARED_VPC_PROJECT_ID">network_project</walkthrough-editor-select-regex>
+to match the project in which your shared VPC exists.
+
+### Update Region
+By default, Forseti deploys into us-central1.  If you deployed Forseti in a different region, you will need to
+update the cloudsql_region, server_region, and client_region input variables.
+
+Update the <walkthrough-editor-select-regex
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  regex="us-central1">cloudsql_region</walkthrough-editor-select-regex>
+to match the region where CloudSQL is deployed.
+
+Update the <walkthrough-editor-select-line
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  startLine=63
+  endLine=63
+  startCharacterOffset=21
+  endCharacterOffset=33>server_region</walkthrough-editor-select-line>
+to match the region where the Forseti Server VM is deployed.
+
+Update the <walkthrough-editor-select-line
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  startLine=64
+  endLine=64
+  startCharacterOffset=21
+  endCharacterOffset=33>client_region</walkthrough-editor-select-line>
+to match the region where the Forseti Client VM is deployed.
+
+## Add Input Variables for Custom Configurations
+Starting with Forseti Security 2.23, Terraform will manage your server
+ configuration file for you.  Configuration options will now be input
+ variables that are defined in the Terraform module.  This will ensure
```

Comment:
We should tell people where to look for the available terraform variables.

Reply:
Done